acceptance/Keywords.robot

```diff
@@ -2,106 +2,29 @@
 Library         String
 Library         Process
 
+
 *** Variables ***
-${TESTED_APP_NAME}  static_test
+#autosleep under brokers-sandboxes
+${SPACE_GUID}	2d745a4b-67e3-4398-986e-2adbcf8f7ec9
+#default
+${SERVICE_NAME}	autosleep
+${PLAN_GUID}	0179eda6-bbcb-408a-aff9-d21d748056dc
+${TESTED_APP_NAME}	static_test
+${TESTED_APP_GUID}  3a4e9275-e937-4735-b272-84ddea21b1f6
```

Comment:
Can't we just use names for APP, PLAN and SPACE? How would an operator retrieve the UUIDs with the CLI?